bitnami/kubeapps/values.yaml

```diff
@@ -1515,7 +1515,7 @@ clusters:
     domain: cluster.local
 ## @param featureFlags [object] Feature flags (used to switch on development features)
 ##
-featureFlags:
+featureFlags: ""
```

Comment:
Wouldn't it be `featureFlags: {}` instead since it is an object? cc @miguelaeh

Reply:
You're totally right

Reply:
ups, yes, my fault